frontend/dockerfile/dockerfile_test.go

```diff
@@ -6297,14 +6321,19 @@ COPY Dockerfile Dockerfile
 
 // moby/buildkit#1301
 func testDockerfileCheckHostname(t *testing.T, sb integration.Sandbox) {
-	integration.SkipOnPlatform(t, "windows")
 	f := getFrontend(t, sb)
-	dockerfile := []byte(`
+	dockerfile := []byte(integration.UnixOrWindows(
+		`
 FROM busybox
 RUN cat /etc/hosts | grep foo
```

Comment:
an equivalent of `/etc/hosts` is perhaps to check the regkey?

```
RUN  reg query HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\Tcpip\Parameters /v Hostname | findstr FOO`
```

Reply:
I've implemented the change.